src/pages/FrontPage.js

```diff
@@ -48,81 +37,15 @@ export const  FrontPage = React.createClass({
-                  Klikk på elev/lærer knappen i navigasjonsmenyen for å skifte modus.
-                  Når du er i lærer-modus vil skoleemner ligge øverst i oppgavefilteret.
-                </div>
-              </div>
-              <div className={styles.infoBoxRow}>
-                <div>
-                  <h3>Lærer</h3>
-                  text text text text text text text text text text text
-                  text text text text text text text text text text text
-                  text text text text text text text text text text text
-                  text text text text text text text text text text text
-                  <br /><br />
-                  <a className={styles.link} href={hrefAssistant}>Lær mer</a>
-                </div>
-                <div>
-                  <h3>Veileder</h3>
-                  text text text text text text text text text text text
-                  text text text text text text text text text text text
-                  text text text text text text text text text text text
-                  text text text text text text text text text text text
-                  <br /><br />
-                  <a className={styles.link} href={hrefTeacher}>Lær mer</a>
-                </div>
-              </div>
-            </div>
-          </Collapse>
+          <TeacherInfobox isStudentMode={this.props.isStudentMode}/>
         </Row>
 
-         <Row>
-          {/*Filter desktop*/}
-          <Col xsHidden sm={4} md={3} lg={2}>
-            <br/>
-            <LessonFilterContainer/>
-          </Col>
-
-          {/*Filter mobile*/}
-          <Col smHidden mdHidden lgHidden xs={12}>
-            <br/>
-            <Button bsStyle='success' onClick={() => this.setState({showMobileFilter: !this.state.showMobileFilter})}>
-              <Glyphicon glyph={this.state.showMobileFilter ? 'chevron-down' : 'chevron-right'}/>
-              Vis/skjul filter
-            </Button>
-            <br/>
-            <br/>
-            <Collapse in={this.state.showMobileFilter}>
-              <div>
-                <LessonFilterContainer/>
-              </div>
-            </Collapse>
-          </Col>
-
-          <Col xs={12} sm={8} md={9} lg={10}>
-            <Row>
-              <Col xs={12}>
-                <h2>Kurs</h2>
-                <CourseList courses={this.props.courses}/>
-              </Col>
-            </Row>
-            {Object.keys(this.props.externalCourses).length > 0 ?
-              <Row>
-                <Col xs={12}>
-                  <h2>Kurs på andre nettsider</h2>
-                  <CourseList courses={this.props.externalCourses}/>
-                </Col>
-              </Row>
-            :null}
-          </Col>
+        {/* Filter and courses */}
+        <Row>
+          <Filter courses={this.props.courses} externalCourses={this.props.externalCourses}/>
+          <Courses courses={this.props.courses} externalCourses={this.props.externalCourses}/>
```

Comment:
Nå er det kun Courses som bruker `this.props.courses` og `this.props.externalCourses`. Blir det bedre om Courses får disse direkte fra selectorene istedet for å gå gjennom FrontPage?